cs/test/core/Core.csproj

```diff
@@ -29,53 +27,23 @@
     <BondOptions>--fields --collection-interfaces</BondOptions>
   </PropertyGroup>
   <ItemGroup>
-    <BondImportDirectory Include="import dir with spaces\" />
+    <BondImportDirectory Include="import dir with spaces" />
```

Comment:
The convention for MSBuild is that things representing directories have the trailing slash.

Reply:
@chwarr I'm getting this error: Invalid project item: "..\core\import dir with spaces\". A file item cannot end with a path separator.

Reply:
Hrm... I wonder where this is coming from now. I cannot find this error message in either the MSBuild source code or the .NET SDK...